src/hpv_query.py

```diff
@@ -71,46 +82,47 @@ def query_json_ids(
     return unique_ids
 
 def process_one_file(
```

Comment:
I think this could probably be simplified as well.
```python
from typing import Iterable


def stream_entries(file: str) -> Iterable:
    file = Path(file)
    with file.open() as in_file:
        for line in in_file:
            yield json.loads(line.strip())


# This is more efficient because you don't have to load
# The entire file, but everything is lazily preprocess
entries = stream_entries(file)
queried_entries = filter(lambda entry: contains_terms(entry["content"]), entries)
# You can use the "a" option to append to a file,
# and then you won't have to load and write out the entries over and over again
with open(save_file, "a") as out_file:
    for entry in entries:
        out_file.write(json.dumps(entry) + "\n")

```